LibExample/CMakeLists.txt

```diff
@@ -59,3 +59,14 @@ else ()
 	target_link_libraries (generate_flow -Wl,--whole-archive minifi-http-curl -Wl,--no-whole-archive)
 endif ()
 
+
+add_executable(monitor_directory monitor_directory.c)
+
+# Link against minifi, yaml-cpp, civetweb-cpp, uuid, openssl and rocksdb
```

Comment:
should we need to link against yaml?  not sure I understand its function in the lib sense.